ansible.cfg

```diff
@@ -19,7 +19,7 @@ remote_user = root
 roles_path = roles/
 gathering = smart
 fact_caching = jsonfile
-fact_caching_connection = $HOME/ansible/facts
+fact_caching_connection = /tmp/ansible/cached_facts
```

Comment:
This should probably stay variablized.

Reply:
Hrm, why would a folder in $HOME be the right place for a cache?  Especially a non-dotted name?